samza-api/src/main/java/org/apache/samza/config/Config.java

```diff
@@ -154,6 +155,9 @@ public double getDouble(String k) {
       return defaultValue;
 
     String value = get(k);
+    if (value.isEmpty()) {
+      return Collections.emptyList();
```

Comment:
Can we use `StringUtils.isNotBlank(value)` instead of `value.isEmpty()`?
The latter doesn't cover `"   "`.